SwiftyExperiments/SwiftyExperiments/Sean/Data/AnnouncementData.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+let announcementData = [Announcement(imageName: "Teammate",
```

Comment:
I'd use string files for this.

Reply:
This is mocked data for time being, so this data is supposed to be dynamic and it's contents will change, I created two or three potential announcements for the time being! Happy if we leave as is? 

I can move 'Learn more' text to a string file though!